gc/base/GCExtensionsBase.cpp

```diff
@@ -136,7 +136,7 @@ MM_GCExtensionsBase::initialize(MM_EnvironmentBase* env)
 
 #if defined(AIXPPC)
 	requestedPageSize = SIXTY_FOUR_KB; /* Use 64K pages for AIX-32 and AIX-64 */
-#elif ((defined(LINUX) || defined(OSX)) && (defined(J9X86) || defined(J9HAMMER)))
+#elif ((defined(LINUX) || defined(OSX)) && (defined(J9X86) || defined(J9HAMMER) || defined(J9RISCV64)))
```

Comment:
Does RISC-V Linux provided 2M Huge page by default?

Reply:
There is no official document on RISC-V Linux (for now we are testing on Fedora_RISCV) as these OS distributions are still under development. So I agree to stick to the default value for the moment till everything becomes clear on OS side in terms of these settings.